Maddori.Apple/Maddori.Apple/Screen/Home/Home/TeamManageSettingCell.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  TeamManageSettingCell.swift
+//  Maddori.Apple
+//
+//  Created by 이성호 on 2023/02/06.
+//
+
+import UIKit
+
+import SnapKit
+
+final class TeamManageSettingCell: BaseTableViewCell {
+    
+    // MARK: - property
+    
+    let cellTitleLabel: UILabel = {
+        let label = UILabel()
+        label.font = .label2
+        label.textColor = .black100
+        return label
+    }()
+    
+    override func render() {
+        self.addSubview(cellTitleLabel)
+        cellTitleLabel.snp.makeConstraints {
+            $0.centerY.equalToSuperview()
+            $0.directionalHorizontalEdges.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+        }
+    }
+    
+    override func configUI() {
+        self.backgroundColor = .white200
+    }
+}
+
+final class TeamManageSettingFooterCell: UITableViewHeaderFooterView {
```

Comment:
p4;

```suggestion
final class TeamManageSettingFooterView: UITableViewHeaderFooterView {
```

Cell이란 네이밍 보단 View어떨까요??

Reply:
좋습니다!

[1ceb7d5](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/303/commits/1ceb7d5464beca5586bdcbc7f9e095aabe82521a)